recipe/meta.yaml

```
@@ -23,25 +23,28 @@ requirements:
     - pytest
     - python >=3.7
   run:
-    - billiard >=3.6.4.0,<4.0
-    - click >=8.0.3,<9.0
-    - click-didyoumean >=0.0.3
+    - billiard >=4.1.0,<5.0
+    - click >=8.1.2,<9.0
+    - click-didyoumean >=0.3.0
     - click-plugins >=1.1.1
     - click-repl >=0.2.0
-    - kombu >=5.2.3,<6.0
-    - importlib-metadata >=1.4.0
+    - kombu >=5.3.0,<6.0
+    - importlib-metadata >=3.6
+    - backports.zoneinfo >=0.2.1
     - python >=3.7
+    - python-tzdata >=2022.7
+    - python-dateutil >=2.8.2
     - pytz >=2021.3
     - vine >=5.0.0,<6.0
 
 test:
   requires:
     - dnspython
     - pymongo
-    - pytest~=6.2
+    - pytest
     - pytest-celery
     - pytest-subtests
-    - pytest-timeout~=1.4.2
+    - pytest-timeout
```

```suggestion
    - pytest-timeout
    - pytest-click
```

looks like the versions are pinned in celery but not here https://github.com/celery/celery/blob/main/requirements/test.txt